docs/changelog.txt

```diff
@@ -52,16 +52,15 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 
 ## API
 - ``Gui::any_civzone_hotkey``, ``Gui::getAnyCivZone``, ``Gui::getSelectedCivZone``:  new functions to operate on the new zone system
-
```

Comment:
I'm not sure why these extra line breaks were in here, and if I should just leave them?

Reply:
the GitHub web UI is a beast to use for resolving merge conflicts on mobile. they might be left over from that. Taking them out is fine. Otherwise I'll clean them up in some future edit.